cmd/humanlog/main.go

```diff
@@ -22,6 +23,7 @@ func main() {
 
 	prefix := rgbterm.FgString(app.Name+"> ", 99, 99, 99)
 
+	log.SetOutput(colorable.NewColorableStdout())
```

Comment:
this should be `colorable.NewColorableStderr()` to avoid mixing the stdout/stderr streams